### PR TITLE
Section

### PR DESCRIPTION
**Pull request by Feruzteame**
This is the section part of web-page. before i start my work i try to classify the page by comment in to header, nav, section , article, aside and footer. I think it help us. Please review and add comment. 
closes #7

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+body{
```
Comment (harunaltunhr) [inline style.css]: Dear Feruz, If you change Body style from here, all elements will affect from this. Maybe we arrange padding and box-shadows at the end

File: index.html [hunk 2]
```diff
@@ -12,6 +12,34 @@
...
+     <p id="intro">
+       This the blue-sky collaborative exercise. It is aimed how to collabrate in git-hub.
+       I find this project very important because it help us in real world project. Codeing 
+       is not how to write your code it include how to communicate with your client, your collaga,
+       and web-designer.
+     </p>
+   </section>
+
+
+<!--Article-->
```
Comment (harunaltunhr) [inline index.html:41]: I think first aside and then article will come, so arranging elements with float will be easier

**Review by harunaltunhr — CHANGES_REQUESTED** [review]
I think you should assign all teams as reviewers. I will do it for you :+1:

**Comment by adekimpianna** [comment]
@Feruzteame I like what you write for the content! 😄 by the way, can you check on issue #12 to discuss about the changes

File: style.css (hunk 1 above)
Comment (Feruzteame) [reply style.css]: i remove it can u check it pls

File: index.html (hunk 2 above)
Comment (Feruzteame) [reply index.html:41]: good point

**Comment by Feruzteame** [comment]
i try to improve my code can u check it pls

**Review by miroslavveljanoski — APPROVED** [review]
This looks ok for me. @Feruzteame resolve the conflicts so we can merge your branch to the master

**Review by adekimpianna — APPROVED** [review]

**Review by harunaltunhr — APPROVED** [review]
It looks well

**Comment by Feruzteame** [comment]
hey guys i try to resolve the conflict. there was a lot of text . i delete it. and i add `background-color : #fafafa`. if you happy merge it. if not add comment. 👍

**Review by harunaltunhr — CHANGES_REQUESTED** [review]
Feruz this is not correct !!! You deleted every thing. You need to fix conflict only. So you should see both master and your codes

**Comment by Feruzteame** [comment]
So sad i made a mistake.

**Pull request closed without merging**